flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/ManuallyTriggeredDirectExecutor.java

```diff
@@ -32,8 +33,17 @@
  */
 public class ManuallyTriggeredDirectExecutor implements Executor {
 
+	protected final Executor executorDelegate;
```

Comment:
Maybe we could get rid of this class completely after refactoring the `FailoverStrategy` tests.

Reply:
Alright, then we keep it of course.